newIDE/app/src/UI/Theme/SolarizedDarkTheme/index.js

```diff
@@ -0,0 +1,231 @@
+// @flow
+import { type Theme } from '../DefaultTheme';
+import { createMuiTheme, darken, lighten } from '@material-ui/core/styles';
+import './Mosaic.css';
+import './EventsSheet.css';
+import 'react-virtualized/styles.css'; // Styles for react-virtualized Table
+import './Table.css';
+import './Markdown.css';
+
+const almostWhite = '#EEE';
+const lightWhite = '#DDD';
+const notSoWhite = '#CCC';
+const solarizedDark1 = '#047AA6';
+const solarizedDark2 = '#047AA6';
```

Comment:
Don't use two varaibles if they are the same

Reply:
okay